librz/analysis/fcn.c

```diff
@@ -2441,6 +2441,7 @@ RZ_API RZ_OWN RzPVector *rz_analysis_function_args(RzAnalysis *a, RzAnalysisFunc
 			// where each var is assigned at its argnum
 			if (argnum >= rz_pvector_len(args)) {
 				if (!rz_pvector_reserve(args, argnum + 1)) {
+					rz_pvector_free(tmp);
```

Comment:
this is ok, though probably `goto err` with err being at line 2454 would be less error prone

Reply:
I plan to do another sweep for memory leaks. I can change this tomorrow then. If there are no objections regarding the semantics, should be fine to merge then.

Reply:
Nevermind, fixed.